synapse/storage/databases/main/group_server.py

```diff
@@ -989,7 +1007,7 @@ def _add_user_to_summary_txn(
                 WHERE group_id = ? AND role_id = ?
             """
             txn.execute(sql, (group_id, role_id))
-            (order,) = txn.fetchone()
+            (order,) = cast(Tuple[Optional[int]], txn.fetchone())
```

Comment:
Same as a comment above: I think order is never NULL?

Reply:
I have done this.